### ISSUE TITLE
Can I use `gpytorch.optim.NGD` with `PredictiveLogLikelihood` loss?

### ISSUE THREAD

**Issue by Alaya-in-Matrix** [cornellius-gp/gpytorch#2069]
When using natural gradient with `VariationalELBO` loss, I can use a relatively large learing rate for variational parameters (0.1 or 1), however, when replacing `VariationalELBO` with `PredictiveLogLikelihoo`, such a large learning rate always results in `NotPSDError`, is there any problem applying `gpytorch.optim.NGD` with `PredictiveLogLikelihood` loss?

**Comment by gpleiss** [comment]
This objective function is not compatible with NGD. Only the VariationalELBO is.
